packages/grid/_modules_/grid/hooks/features/rows/useGridEditRows.ts

```diff
@@ -315,48 +274,58 @@ export function useGridEditRows(apiRef: GridApiRef) {
     [apiRef],
   );
 
-  const handleDoubleClick = React.useCallback(
+  const handleCellEditExit = React.useCallback(
+    (params: GridCellParams, event?: React.SyntheticEvent) => {
+      // TODO check if its propagation was stopped
```

Comment:
To be clearer, the event would continue to propagate

```suggestion
      // TODO check if event internal handling was prevented
```

Reply:
No, I was meaning to add the following code:

```ts
if (!params.isEditable || event.isPropagationStopped()) {
  return;
}
```

`handleCellEditEnter` already has it but this method doesn't. However, with #1403 all of this manual checks will go away.